website/source/docs/glossary.html.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: "docs"
+page_title: "Consul Glossary"
+sidebar_current: "docs-glossary"
+description: |-
+  This page collects brief definitions of some of the technical terms used in the documentation.
+---
+
+# Consul Glossary
+
+This page collects brief definitions of some of the technical terms used in the documentation for Consul and Consul Enterprise, as well as some terms that come up frequently in conversations throughout the Consul community.
+
+* [Agent](#agent) 
```

Comment:
I don't think this page is long enough to hotlink each section. In the preview, the bullets are spaced far apart so the ToC makes the page longer.

Reply:
I think hotlinks are nice since we don't know where the page fold is located. Also, hopefully, the list will be long enough someday to need them.